src/hammer-vlsi/hammer_vlsi/hammer_build_systems.py

```diff
@@ -36,12 +36,18 @@ def build_makefile(driver: HammerDriver, append_error_func: Callable[[str], None
         - par
         - drc
         - lvs
+        - sim
```

Comment:
should these sim actions be split into `sim-par` and `sim-syn`?

Reply:
I think we should label them since its more important that there are targets for these althouh technically there is only one `sim` action, and its just called with different inputs.

Reply:
Fixed.